docs/src/environments/restore.md

```diff
@@ -0,0 +1,108 @@
+---
+title: Restore an environment from a backup
+sidebarTitle: Restore environments
+description: See how to restore an environment from a previous state.
+weight: -10
+---
+
+Once you have [backups of your environment](./backup.md), you might want to restore data from a previous point.
+
+To restore an environment, you need an [Admin role for that environment type](../administration/users.md).
```

Comment:
If it were the same paragraph, I would like this suggestion. But I think it should be separate because of the content and in that case I think it helps to make it clear.